### PR TITLE
Carbon 483 - New users and organisations added

### PR DESCRIPTION
**Pull request by tharindugayanga**

### CODE REVIEW

File: web/src/Components/Programme/AddNewProgramme/programmeCreationComponent.tsx [hunk 1]
```diff
@@ -106,10 +106,10 @@ export const ProgrammeCreationComponent = (props: any) => {
   const [projectLocationMapOutlineLayer, setProjectLocationMapOutlineLayer] = useState<any>();
   const [projectLocationMapCenter, setProjectLocationMapCenter] = useState<number[]>([]);
 
-  const mapType = process.env.REACT_APP_MAP_TYPE ? process.env.REACT_APP_MAP_TYPE : 'None';
+  const mapType = process.env.REACT_APP_MAP_TYPE ? process.env.REACT_APP_MAP_TYPE : 'Mapbox';
```
Comment (leshanw) [inline web/src/Components/Programme/AddNewProgramme/programmeCreationComponent.tsx:109]: remove the token and map type

File: backend/services/src/national-api/national.api.module.ts [hunk 2]
```diff
@@ -34,14 +37,16 @@ import configuration from 'src/configuration';
...
+    LocationModule
   ],
   controllers: [ 
 		NationalAPIController, 
 		UserController, 
 		AuthController, 
 		CompanyController, 
 		ProgrammeController, 
-		SettingsController, 
+		SettingsController,
```
Comment (leshanw) [inline backend/services/src/national-api/national.api.module.ts:48]: format the file

File: web/src/Components/Programme/AddNewProgramme/programmeCreationComponent.tsx (hunk 1 above)
Comment (tharindugayanga) [reply web/src/Components/Programme/AddNewProgramme/programmeCreationComponent.tsx]: Fixed

File: backend/services/src/national-api/national.api.module.ts (hunk 2 above)
Comment (tharindugayanga) [reply backend/services/src/national-api/national.api.module.ts]: Fixed

**Review by leshanw — APPROVED** [review]

**Pull request merged**